zello-sdk-sample-contacts/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
 	}
 
     dependencies {
```

Comment:
looks like the indentations are off between this and `repositories` above

Reply:
Right, AS has gone through so many default formatting styles that it's now all over the place. Re-formatted them all, but that also lit up a lot of changes.